api/HTMLInputElement.json

```
@@ -244,7 +244,7 @@
               "version_added": false
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "25"
```

The commit date was 2012-05-10 and using https://www.chromium.org/developers/calendar that maps to Chrome 21:

```suggestion
              "version_added": "21"
```

---

Ah great, thanks for tracking this down!  There was no Chrome release in between 18 and 25, so if the commit maps to Chrome 21, we'd bump it to 25.

---

Oh, right :)